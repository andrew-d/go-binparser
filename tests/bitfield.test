[
    {"name": "FooStruct",
     "fields": [
        {"name": "bits", "offset": 0, "type": "bitfield.uint8", "bit_fields": [
            {"name": "flag1"},
            {"name": "not_a_flag", "size": 2},
            {"name": "remaining", "size": 5}
        ]}
     ]}
]
