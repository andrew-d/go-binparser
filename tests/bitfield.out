Flag1,%!s(uint8=1)
Not_A_Flag,%!s(uint8=2)
Remaining,%!s(uint8=21)
